introduction.md

```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu pellentesque leo. Curabitur blandit sapien at feugiat viverra. Donec sed lorem eu mauris efficitur molestie. Mauris tristique arcu eget mauris sodales ultricies. Maecenas at consectetur nisi. Sed ac mi vitae mi lobortis ultrices. Mauris dapibus tincidunt ipsum, ut mollis turpis dapibus non. Aliquam nec tincidunt ipsum. Fusce imperdiet justo sed massa placerat, at imperdiet quam dapibus.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.